src/Links.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {Button} from './Button';
+
+
+export function Links(props){
+    const buttons = props.links.map(({label, to}) => <Button key={label} name={label} to={to} />);
+    return(
+        <div className="navbar a:hover">
```

Comment:
Don't use `a:hover`.  You can use `:hover` on a class, like `.my-navbar:hover`, and it will automatically apply those styles when the element is hovered over.